aspnet/web-pages/overview/releases/aspnet-web-pages-2-developer-preview-readme.md

```diff
@@ -25,7 +25,7 @@ To install Web Pages 2 Developer Preview, you have these options:
 
 - Install WebMatrix 2 Beta using the [Web Platform Installer](https://go.microsoft.com/fwlink/?LinkId=226883). WebMatrix is a set of free web development tools that includes ASP.NET Web Pages. For more information, see the installation section in [The Top Features in ASP.NET Web Pages 2 Developer Preview](https://go.microsoft.com/fwlink/?LinkID=227824).
 
-- Install Web Pages 2 Developer Preview directly using the [download link](https://go.microsoft.com/fwlink/?LinkID=226335). Use this approach if you want to create Web Pages applications using a text editor such as Notepad. In order to run Web Pages 2 applications, you must have IIS Express 7.5. (This is included automatically with WebMatrix.) For tips on how to test a Web Pages page using IIS Express, see the sidebar "Creating and Testing ASP.NET Pages Using Your Own Text Editor" in [Getting Started with WebMatrix and ASP.NET Web Pages](https://go.microsoft.com/fwlink/?LinkId=202889).
+- Install Web Pages 2 Developer Preview directly using the [download link](https://www.microsoft.com/en-us/download/details.aspx?id=34600). Use this approach if you want to create Web Pages applications using a text editor such as Notepad. In order to run Web Pages 2 applications, you must have IIS Express 7.5. (This is included automatically with WebMatrix.) For tips on how to test a Web Pages page using IIS Express, see the sidebar "Creating and Testing ASP.NET Pages Using Your Own Text Editor" in [Getting Started with WebMatrix and ASP.NET Web Pages](https://go.microsoft.com/fwlink/?LinkId=202889).
```

Comment:
We need to avoid hardcoded locales, so the site can auto-direct based on the browser's locale setting:
```suggestion
- Install Web Pages 2 Developer Preview directly using the [download link](https://www.microsoft.com/download/details.aspx?id=34600). Use this approach if you want to create Web Pages applications using a text editor such as Notepad. In order to run Web Pages 2 applications, you must have IIS Express 7.5. (This is included automatically with WebMatrix.) For tips on how to test a Web Pages page using IIS Express, see the sidebar "Creating and Testing ASP.NET Pages Using Your Own Text Editor" in [Getting Started with WebMatrix and ASP.NET Web Pages](https://go.microsoft.com/fwlink/?LinkId=202889).
```